.circleci/config.yml

```diff
@@ -134,7 +138,7 @@ commands:
         type: string
       split:
```

Comment:
can we do away with split (and the environment variable)?

Reply:
done!